org-fc-review.el

```diff
@@ -94,15 +94,15 @@ Valid contexts:
       (when (yes-or-no-p "Flashcards are already being reviewed. Resume? ")
         (org-fc-review-resume))
     (let* ((index (org-fc-index context))
-           (cards (org-fc-index-filter-due index)))
-      (if org-fc-shuffle-positions
-          (setq cards (org-fc-index-shuffled-positions cards))
-        (setq cards (org-fc-index-positions cards)))
-      (if (null cards)
+           (cards (org-fc-index-filter-due index))
+           (positions (if org-fc-shuffle-positions
+                          (org-fc-index-shuffled-positions cards)
+                        (org-fc-index-positions cards))))
+      (if (null positions)
           (message "No cards due right now")
```

Comment:
Should the error message refer to it as positions, too?

Reply:
You're right. In fact, there are quite a few other places here (e.g. `org-fc-review-next-card`) which actually refer to positions. However, I think it would be easier to address those as a part of the work resulting from #94 so I'll leave them alone for now.